react/Header/UserAccount/UserAccount.less

```diff
@@ -65,15 +65,16 @@
   bottom: 0;
 }
 
-.userDisplayName {
+.userName {
   pointer-events: none;
   overflow: hidden;
   text-overflow: ellipsis;
   display: inline-block;
-  max-width: 101px; // ~ 10 characters
+  white-space: nowrap;
+  max-width: 100px; // ~ 10 characters
 
   @media only screen and (min-width: 400px) {
```

Comment:
That is totally OK so long as you ritually apologise for incurring design debt :trollface: